staging/src/k8s.io/apiserver/pkg/server/options/serving.go

```diff
@@ -108,7 +108,7 @@ func NewSecureServingOptions() *SecureServingOptions {
 }
 
 func (s *SecureServingOptions) DefaultExternalAddress() (net.IP, error) {
-	if !s.ExternalAddress.IsUnspecified() {
+	if s.ExternalAddress != nil && !s.ExternalAddress.IsUnspecified() {
```

Comment:
didnt see an issue, whats ur binding address?

[https://github.com/kubernetes/kubernetes/blob/7c112762b0df87f3a1772f3bbede7fe9185fa249/staging/src/k8s.io/apimachinery/pkg/util/net/interface.go#L408](https://github.com/kubernetes/kubernetes/blob/7c112762b0df87f3a1772f3bbede7fe9185fa249/staging/src/k8s.io/apimachinery/pkg/util/net/interface.go#L408)

Reply:
if `ExternalAddress` is nil, `DefaultExternalAddress` will just return nil, not goto `ChooseBindAddress` func. 

you can find `How to reproduce it` in https://github.com/kubernetes/kubernetes/issues/72404